Testreglar/125/App/app-1.2.5a.json

```diff
@@ -0,0 +1,186 @@
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Viktig visuell informasjon i videoen er tilstrekkelig beskrevet, i rett rekkefølge i synstolket lydspor."
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Viktig visuell informasjon i videoen er ikke tilstrekkelig beskrevet, i rett rekkefølge i synstolket lydspor."
+				}
+			}
+		},
+		{
+			"stegnr": "3.7",
+			"spm": "Er viktig visuell informasjon formidlet via den originale lyden i videoen?",
+			"ht": "<p>Hva som er «viktig» visuell informasjon vil variere etter type video, og blir avgjort etter en totalvurdering. Eksempler på visuell informasjon som vanligvis skal synstolkes er ifølge merknad til definisjonen av synstolking informasjon om</p>\n<ul>\n<li>handling (hva)</li>\n<li>karakterer/personer (hvem)</li>\n<li>sceneskifter/stedskifter (hvor, hvordan)</li>\n<li>tekst på skjermen og</li>\n<li>annet viktig visuelt innhold som endres underveis</li>\n</ul>\n<p>Aktuelle metoder for å finne ut dette er: </p>\n<ul>\n<li>starte videoklippet og vurder de stikkprøvene du registrerte i forrige steg</li>\n<li>sammenlign informasjon formidlet i original lyden med visuelle innholdet i videoen</li>\n<li>vurder om viktig visuell informasjon i videoen er tilstrekkelig beskrevet, i rett rekkefølge via den originale lyden </li>\n</ul>\n<p><strong>Merk:</strong> Med \"tilstrekkelig beskrevet\" menes det at brukeren skal få beskrevet meningsbærende visuell informasjon, slik at en kan forstå videoens sammenhenger, fortellinger og budskap gjennom lyden. I tillegg vil det være nyttig å ta høyde for hvem, hva og hvor. </p>\n<p>Det er i utgangspunktet ikke nødvendig med synstolking av en video der den visuelle informasjonen er statisk, for eksempel der den som snakker står foran et lerret med logoen til en virksomhet, dersom det ellers ikke brukes andre viktige visuelle virkemidler. Det kan beskrives muntlig innledningsvis eller gis som programmatisk bestemt tekstalternativ, jamfør teknikk G203.</p>",
```

Comment:
Det bør legges til egne steg for å ivareta G203. I tillegg bør det presiseres hva som er ok med hensyn til at videoen må pauses for å få med seg synstolkingen.